Build docs is failing due to that the action isn't able to find `pyproject.toml`

Build `pymoors` doc is failing with in the CI

```sh
Run uv sync --group docs
error: No `pyproject.toml` found in current directory or any parent directory
Error: Process completed with exit code 2.
```

--- a/.github/workflows/deploy-docs.yaml
+++ b/.github/workflows/deploy-docs.yaml
@@ -29,10 +29,7 @@ jobs:
           python-version: ${{ env.DEFAULT_PYTHON }}
 
       - name: Install pymoors
-        run: |
-          uv sync --group docs
-          make build-dev
+        run: make pymoors-build-docs
 
       - name: Deploy to GitHub Pages
-        run: |
-          mkdocs gh-deploy --force
+        run: mkdocs gh-deploy --force

